packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -102,7 +101,7 @@ struct CreateDataTypeRequest {
     #[schema(value_type = VAR_DATA_TYPE)]
     schema: serde_json::Value,
     owned_by_id: OwnedById,
-    actor_id: CreatedById,
+    actor_id: UpdatedById,
```

Comment:
Not suggesting we change it now or anything, but the name `UpdatedById` seems to be slightly misleading now, as it also means creating on the very first entity edition. I guess we can consider creation an "update" essentially. Perhaps `TouchedById` would capture the meaning better, not sure

Reply:
I think creation is an update, but I appreciate in CRUD there's a semantic difference so perhaps we do want something else. We could possibly name it `editionCreatedBy` 🤷

Reply:
I like `editionCreatedBy`, but I think this is better suited for another PR 🙂